pubsub/pullstream.go

```diff
@@ -47,10 +47,16 @@ func newPullStream(ctx context.Context, streamingPull streamingPullFunc, subName
 			spc, err := streamingPull(ctx, gax.WithGRPCOptions(grpc.MaxCallRecvMsgSize(maxSendRecvBytes)))
 			if err == nil {
 				recordStat(ctx, StreamRequestCount, 1)
+				streamAckDeadline := int32(maxDurationPerLeaseExtension / time.Second)
+				// By default, maxDurationPerLeaseExtension, aka MaxExtensionPeriod, is disabled,
+				// so in these cases, use a healthy default of 60 seconds.
+				if streamAckDeadline <= 0 {
+					streamAckDeadline = 60
+				}
 				err = spc.Send(&pb.StreamingPullRequest{
 					Subscription: subName,
 					// We modack messages when we receive them, so this value doesn't matter too much.
```

Comment:
Nit: perhaps remove/reword this sentence, since it does seem to matter somewhat :)